apollo-router/src/lib.rs

```diff
@@ -90,6 +90,7 @@ pub use crate::context::Context;
 pub use crate::executable::main;
 pub use crate::executable::Executable;
 pub use crate::notification::Notify;
+pub use crate::plugins::telemetry::otel;
```

Comment:
why is it public?

Reply:
Hmm I think that's a mistake, I wanted to get rid of tracing_opentelemetry even for tests but for now I'll just keep it, maybe I'll remove it later but that's ok. the otel module is no longer pub. Good call